### PR TITLE
RM - Messaging Center infrastructure

### PR DESCRIPTION
**Pull request by anvmn**
#602

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2023-01-16 15-52

**Comment by anvmn** [comment]
@mariano-dagostino please review

File: client/src/elm/Pages/MessagingCenter/Model.elm [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import AssocList exposing (Dict)
+import Backend.Entities exposing (..)
+import Backend.Measurement.Model
+import EverySet exposing (EverySet)
+import Gizra.NominalDate exposing (NominalDate)
+import Pages.Page exposing (Page)
+
+
+type alias Model =
+    { -- filter : Maybe CaseManagementFilter
```
Comment (anvmn) [inline client/src/elm/Pages/MessagingCenter/Model.elm:12]: Will be used in follow up PRs

File: client/src/elm/Pages/MessagingCenter/Utils.elm [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import Backend.Measurement.Model
+import Backend.Model exposing (ModelIndexedDb)
+import Date exposing (Unit(..))
+import EverySet exposing (EverySet)
+import Gizra.NominalDate exposing (NominalDate, diffDays)
+import Pages.MessagingCenter.Model exposing (..)
+import RemoteData exposing (RemoteData(..), WebData)
+
+
+zzz =
```
Comment (mariano-dagostino) [inline client/src/elm/Pages/MessagingCenter/Utils.elm:14]: @anvmn ?

**Review by mariano-dagostino — APPROVED** [review]
@anvmn `zzz` does not make sense to me...

**Pull request merged**